src/app/public/modules/demo/demo-control-panel.component.html

```diff
@@ -8,6 +8,7 @@
       <button
         class="sky-btn sky-btn-link"
         type="button"
+        [attr.data-test-selector]="'sky-docs-demo-control-panel-reset-button'"
```

Comment:
Reasoning behind the `data-` attribute: https://www.techiediaries.com/add-access-html-data-attribute-angular/